Tests/UnitTests/Purchasing/CustomerInfoTests.swift

```diff
@@ -116,13 +127,14 @@ class BasicCustomerInfoTests: TestCase {
     }
 
     func testListActiveSubscriptions() {
-        expect(self.customerInfo.activeSubscriptions) == ["onemonth_freetrial"]
+        expect(self.customerInfo.activeSubscriptions) == ["onemonth_freetrial", "gold:monthly"]
     }
 
     func testAllPurchasedProductIdentifier() {
         let allPurchased = self.customerInfo.allPurchasedProductIdentifiers
 
-        expect(allPurchased) == ["onemonth", "onemonth_freetrial", "threemonth_freetrial", "onetime_purchase"]
+        expect(allPurchased) == ["onemonth", "onemonth_freetrial",
+                                 "threemonth_freetrial", "gold:monthly", "onetime_purchase"]
```

Comment:
But the more I think about it, I do worry about breaking the semantics of `activeSubscriptions` and `allPurchasedProductIdentifiers`. They'd no longer be "product identifiers" as their documented to be.

Reply:
So I wonder if the solution would be to have a separate property instead that combines both. At the end of the day this is something that users of the iOS SDK won't care about, only hybrids, right?

Reply:
Yeah yeah, this is generally something that hybrids or people using native iOS and Android will care about. I'd _ideally_ like this to be modeled in a new way in somehow in a future version of the SDK because I don't really love this 😛 But 🤷‍♂️